bin/ln

```diff
@@ -75,16 +75,20 @@ unless (@ARGV) {
 }
 exit $rc;
 
+sub VERSION_MESSAGE {
+    print "$Program version $VERSION\n";
+    exit EX_SUCCESS;
+}
+
 sub usage {
-    warn "$Program (Perl bin utils) $VERSION\n";
     warn "usage: $Program [-sf] source_file [target_file | [source_files ...] target_directory]\n";
     exit EX_FAILURE;
 }
 
 sub dolink {
-    my ($file, $target) = @_;
-    return symlink($file, $target) if $options{'s'};
-    return link($file, $target);
+    my ($file, $targ) = @_;
```

Comment:
Is there a reason for changing this variable name? I think I prefer `$target`.

Reply:
perlcritic identified that a global variable had the same name, so renaming the dolink() param was to avoid confusion